Libs/MRML/Core/Testing/vtkMRMLTransformNodeTest1.cxx

```diff
@@ -10,10 +10,43 @@
 
 =========================================================================auto=*/
 
+#include "vtkMRMLBSplineTransformNode.h"
 #include "vtkMRMLCoreTestingMacros.h"
+#include "vtkMRMLScene.h"
 #include "vtkMRMLTransformNode.h"
 
 #include <vtkGeneralTransform.h>
+#include <vtkMatrix4x4.h>
+#include <vtkNew.h>
+#include <vtkTransform.h>
+
+bool Matrix4x4AreEqual(vtkMatrix4x4 *m1, vtkMatrix4x4 *m2)
```

Comment:
That one does not have a tolerance in the comparison, so I'm not sure how well it would work. It would make sense to add some tolerance in vtkMRMLSliceNode::Matrix4x4AreEqual(), too, but I would hesitate to do that in this commit.